.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -27,6 +27,8 @@ If there's no env.py, you are using an out of date version of OpenLane and shoul
 * Upload a tarball containing the relevant design.
 * List the commands used to run the design.
 
+If you see a message like `Reproducible packaged: Please tarball and upload <PATH> if you're going to submit an issue` in your logs, please also tarball and include that path. This will greatly speed up the fixing process.
```

Comment:
When would such a message not be generated?  In that case they should run or_issue manually.

Reply:
If the failure is not OpenROAD-related.